specs/commandline/blob.md

```diff
@@ -29,11 +29,11 @@ Flags:
 Produce a detached signature for a given blob
 
 Usage:
-  notation blob sign [flags] <blob_path>
+  notation blob sign [flags] -n my-blob-signature <blob_path>
 
 Flags:
-  -s,  --signature path             output location of the detached signature
-       --media-type string          media type of the blob. If not provided, notation uses `application/octet-stream`
+  -n,  --signature-name string      friendly name for the detached signature. Signature file will be written to the currently working directory with this name plus ".sig" plus signature format as the file extension
```

Comment:
Is `.sig` necessary? Can it be `mySignature.jws` or `mySignature.cose`?

Reply:
Not necessary but good to have to indicate that it is a signature file. Not all users of notation know that notation uses jws or cose. If we do not add `.sig` and user doesn't know anything about jws/cose (which the most of the userbase is), then it can be confusing as to why notation created the files with jws/cose extension.